config/routes.rb

```diff
@@ -9,6 +9,9 @@
   # get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :likes, only: [:create]
+      resources :comments, only: [:create]
```

Comment:
- [ ] Hi @ErikStoupignan, congrats :tada: you are doing so well in this project. Let's kindly adjust his little issue. There should be a pair of routes for creating comments.
```ruby
  POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```
This requirement is essential for the approval :heavy_check_mark:  of this milestone.

Reply:
Thanks for going through the topic with me on Slack my friend.

I'll add the method "new" in the comments controller as you recommended. (Although it is not a project requirement but to improve the organization)